gateway/bucketfs.go

```diff
@@ -47,22 +49,38 @@ func serveBucket(fs serveBucketFileSystem) gin.HandlerFunc {
 			} else {
 				c.Abort()
 			}
+		} else if target != "" {
+			c.Writer.WriteHeader(http.StatusOK)
+			c.Writer.Header().Set("Content-Type", "text/html")
```

Comment:
just renders a directory's default file if it exists. target should equal `index.html` for now

Reply:
would, `ctype := mime.TypeByExtension(filepath.Ext(c.Request.URL.Path))` not work here?

Reply:
yeah maybe, it would actually be `mime.TypeByExtension(filepath.Ext(fmt.Printf("%s/%s", c.Request.URL.Path, target)))` or similar though

Reply:
Nice, should we do that? Just to max out on flex?

Reply:
yep! will do

Reply:
k, done